TLM/TLM/UI/SubTools/SpeedLimits/Overlay/OverlaySegmentSpeedlimitHandle.cs

```diff
@@ -122,20 +122,21 @@
             }
         }
 
-        internal static void SetDefaultSpeedLimit(ushort segmentId, NetInfo netInfo, SetSpeedLimitAction action) {
+        internal static void SetDefaultSpeedLimit(ushort segmentId,
+                                                  [NotNull] NetInfo netInfo,
+                                                  SetSpeedLimitAction action) {
             switch (action.Type) {
                 case SetSpeedLimitAction.ActionType.SetOverride:
                 case SetSpeedLimitAction.ActionType.Unlimited:
-                    SpeedValue value = action.GuardedValue.Override;
                     bool displayMph = GlobalConfig.Instance.Main.DisplaySpeedLimitsMph;
+                    SpeedValue value = action.GuardedValue.Override;
                     Log.Info($"Setting speed limit for netinfo '{netInfo.name}' seg={segmentId} to={value.FormatStr(displayMph)}");
```

Comment:
Should this logging be limited to debug builds? Also, should it log "Setting _default_ speed limit..."

Reply:
The idea was to have user actions logged to make the log file more useful. I am fine to keep this debug only, will fix.